dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -805,9 +825,15 @@ Simulation::simulate() {
 
   bool criteriaChecked = true;
   try {
-    if (data_ && (exportIIDM_ || activateCriteria_)) {  // no need to update state variable if the IIDM final state is not exported (same for criteria check)
+    // update state variable only if the IIDM final state is exported, or criteria is checked, or lost equipments are exported
+    if (data_ && (exportIIDM_ || activateCriteria_ || (exportLostEquipmentsMode_ != EXPORT_LOSTEQUIPMENTS_NONE))) {
```

Comment:
please add a method for this comparison used several times "exportLostEquipmentsMode_ != EXPORT_LOSTEQUIPMENTS_NONE"

Reply:
Done